features/admin.claims.v1/components/edit/local-claim/edit-basic-details-local-claims.tsx

```diff
@@ -527,11 +601,28 @@ export const EditBasicDetailsLocalClaims: FunctionComponent<EditBasicDetailsLoca
     const onSubmit = (values: Record<string, unknown>) => {
         let data: Claim;
 
+        if (dataType === DataType.COMPLEX && subAttributes.length === 0) {
+            dispatch(
+                addAlert({
+                    description: t("claims:local.forms.subAttributes.validationError"),
+                    level: AlertLevels.WARNING,
```

Comment:
for errors, lets use the error level.

```suggestion
                    level: AlertLevels.ERROR,
```

Reply:
Fixed by - https://github.com/wso2/identity-apps/pull/8111/commits/fc575885da12bc341fd55470310378ecbc3dc6ad